spec/pubid_ieee/identifier_spec.rb

```diff
@@ -368,4 +375,53 @@
 
     it_behaves_like "converts pubid to pubid"
   end
+
+  context "IEEE No 323, April 1971" do
+    let(:original) { "IEEE No 323, April 1971" }
+    let(:pubid) { "IEEE 323-1971" }
+
+    it_behaves_like "converts pubid to pubid"
+  end
+
+  context "IEEE P15026-2, April 2011" do
+    let(:original) { "IEEE P15026-2, April 2011" }
+    let(:pubid) { "IEEE P15026.2-2011" }
```

Comment:
Why is `IEEE P15026-2` converted into `IEEE P15026.2-2011`?

Reply:
I had an impression that most of IEEE identifiers have format: IEEE {number}.{part}-{year}
```
ANSI N42.35-2016
IEEE Std C37.20.1-2015
IEEE Std 1900.1-2019
IEEE Std P802.11a/D7
```

or IEEE {number}-{year}
`IEEE Std 515-2011`

As I can see now it's now always like that:
```
IEEE Std 11073-10420-2020
IEEE P11073-10101a/D1, April 2015
IEC/IEEE P60076-57-129_D26
```

For IEEE document numbers starting with "P" when it's 4 digits after it's always using "." before next number
`IEEE P1666.1/D4, October 2015`
Only exception when it's year `IEEE P2410-2020/D5, December 2020`
For "P" and 5 digits it's always using "-" exception only for a few identifiers:
```
IEEE P15288.1/D3.0, June 2014
IEEE P15288.1/D4.0, August 2014
IEEE P15288.1/D4.1, September 2014
IEEE P15288.2/D5.1, August 2014
IEEE P15288.2/D5.2, September 2014
```

I suspect most of cases when number and part using "-" between it's coming from ISO format.

It's not complete research, I just pick random identifiers and investigate particularly numbers with "P" prefix.